### PR TITLE
enh(doc): Update documentation to Centreon 21.04

### PR DESCRIPTION
**Pull request by lpinsivy**
## Description

Update documentation to Centreon 21.04

## Target serie

- [ ] 20.04.x
- [ ] 20.10.x
- [x] 21.04.x (master)

### CODE REVIEW

File: en/installation/introduction.md [hunk 1]
```diff
@@ -23,8 +23,9 @@ Before installation
 3.  [Download Centreon](https://download.centreon.com/)
 4.  Finally, you can install the platform.
 
-To quickly test Centreon from a CentOS or Red Hat 7.x, you can run the following command as **root**:
+To quickly test Centreon from a CentOS / Oracle Linux / RHEL in version 8, you
+can run the following command as **root**:
 
 ```Bash
-curl -L https://raw.githubusercontent.com/centreon/centreon/20.10.x/unattended.sh | sh
+curl -L https://raw.githubusercontent.com/centreon/centreon/master/unattended.sh | sh
```
[the remark this answers is not in the document]
Comment (lpinsivy) [reply en/installation/introduction.md:30]: But when the 21.04 will be released, I dont think so we will directly create 21.04.x branch

File: en/update/update-centreon-platform.md [hunk 2]
```diff
@@ -1,9 +1,9 @@
 ---
 id: update-centreon-platform
-title: Update a Centreon 20.10 platform
+title: Update a Centreon 21.04 platform
 ---
 
-This chapter describes how to update your Centreon 20.10 platform.
+This chapter describes how to update your Centreon 21.04 platform.
```
[the remark this answers is not in the document]
Comment (lpinsivy) [reply en/update/update-centreon-platform.md:6]: Now this chapter will hep users to update a 21.04.x branch, not to upgrade from 20.10.x to 21.04.x

File: en/update/update-centreon-platform.md [hunk 3]
```diff
@@ -1,9 +1,9 @@
 ---
 id: update-centreon-platform
-title: Update a Centreon 20.10 platform
+title: Update a Centreon 21.04 platform
```
[the remark this answers is not in the document]
Comment (lpinsivy) [reply en/update/update-centreon-platform.md:3]: No this chapter will hep users to update a 21.04.x branch, not to upgrade from 20.10.x to 21.04.x

File: en/installation/introduction.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (adr-mo) [reply en/installation/introduction.md:30]: 👍

File: en/update/update-centreon-platform.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (adr-mo) [reply en/update/update-centreon-platform.md:6]: 👍

**Review by adr-mo — APPROVED** [review]

**Pull request merged**